src/typechecker/environment.lisp

```diff
@@ -452,6 +456,7 @@
             :arity 0
             :constructs 'coalton:Boolean
             :classname 'coalton::Boolean/True
+            :docstring "`True` is equivalent to `cl:t`."
```

Comment:
Maybe instead of mentioning `cl` here, the boolean strings could just be 'Boolean true' 'Boolean false' - semantics of the lisp representation could be discussed in the interop documentation.

Consider replacing 'X stands in for Y' and 'X represents Y' with 'The type of Y'

Reply:
otherwise, looks fine to me